infrastructure/demo.tfvars

```diff
@@ -11,3 +11,6 @@ idam_token_jwk_set_uri = "https://idam-api.demo.platform.hmcts.net/o/jwks"
 gateway_url = "https://gateway-ccd.demo.platform.hmcts.net"
 feature_toggle_robotics_case_number_notification_enabled = "true"
 feature_toggle_robotics_support_api_enabled = "true"
+
+managed_identity_object_id="f4243afd-7d84-485d-81f7-c32759f238b7"
+enable_ase = false
```

Comment:
We probably should clean up a bit mor eterraform code but I am happy to have this in separate PR.